modules/compute/event-hub/README.md

```diff
@@ -41,7 +41,7 @@ param clusterName string
 param clusterCapacity int
 param eventHubNamespaces object
 
-module eventhubnamespace 'br/public:compute/event-hub:0.0.1' = {
+module eventhubnamespace 'br/public:compute/event-hub:0.1.0' = {
```

Comment:
```suggestion
module eventhubnamespace 'br/public:compute/event-hub:3.0.1' = {
```

Reply:
Can we update the version numbers for the examples below as well?

Reply:
This commit is just for cosmosdb. I would undo this change.

Reply:
Ah missed that. Makes sense.